filter.go

```diff
@@ -60,31 +83,59 @@ func (filter *Filter) Validate() error {
 // DefaultFilter returns the default filter used by the Matrix server if no filter is provided in the request
 func DefaultFilter() Filter {
 	return Filter{
-		AccountData: DefaultFilterPart(),
+		AccountData: DefaultEventFilter(),
 		EventFields: nil,
 		EventFormat: "client",
-		Presence:    DefaultFilterPart(),
+		Presence:    DefaultEventFilter(),
 		Room: RoomFilter{
-			AccountData:  DefaultFilterPart(),
-			Ephemeral:    DefaultFilterPart(),
+			AccountData:  DefaultRoomEventFilter(),
+			Ephemeral:    DefaultRoomEventFilter(),
 			IncludeLeave: false,
 			NotRooms:     nil,
 			Rooms:        nil,
-			State:        DefaultFilterPart(),
-			Timeline:     DefaultFilterPart(),
+			State:        DefaultStateFilter(),
+			Timeline:     DefaultRoomEventFilter(),
 		},
 	}
 }
 
-// DefaultFilterPart returns the default filter part used by the Matrix server if no filter is provided in the request
-func DefaultFilterPart() FilterPart {
-	return FilterPart{
-		NotRooms:   nil,
-		Rooms:      nil,
+// DefaultEventFilter returns the default event filter used by the Matrix server if no filter is provided in the request
+func DefaultEventFilter() EventFilter {
+	return EventFilter{
 		Limit:      20,
 		NotSenders: nil,
 		NotTypes:   nil,
 		Senders:    nil,
 		Types:      nil,
 	}
 }
+
+// DefaultStateFilter returns the default state event filter used by the Matrix server if no filter is provided in the request
```

Comment:
More of a nitpick, but this comment is quite long, a bit too much for a single line to my liking, could you please wrap it to a more sensible line length (80-90)?